src/python/other-bots/chromium-tests-syncer/run.py

```diff
@@ -36,8 +36,10 @@
 from clusterfuzz._internal.system import environment
 from clusterfuzz._internal.system import shell
 
+ENGINE_FUZZER_NAMES = ['afl', 'centipede', 'googlefuzztest', 'libFuzzer']
```

Comment:
Can you get the list from here: https://github.com/google/clusterfuzz/blob/ad6131e9d61cdbe7838d26b55d82200faeddf84f/src/clusterfuzz/_internal/cron/fuzzer_and_job_weights.py#L54